ITensorGaussianMPS/src/gmps.jl

```diff
@@ -1,5 +1,5 @@
 import Base: sortperm, size, length, eltype, conj, transpose, copy, *
-using ITensors: alias
+using ITensors: alias, ITensorMPS
```

Comment:
```suggestion
using ITensors.ITensorMPS: ITensorMPS
using ITensors: alias
```
related to my comment here: https://github.com/mtfishman/ITensorChemistry.jl/pull/9/files#r1537556024